solved_exercises/dani/02-tuples.py

```diff
@@ -0,0 +1,43 @@
+import itertools
+'''*********************************************************************************************'''
+print('*****************Check wether an element exists within a tuple******************')
+my_tuple = 1, 2, 3, 3, 4, 5, 6
+print('my_tuple is', my_tuple)
+a=3
+b=7
+print('Version1')
+print (a in my_tuple and (a," is in my_tuple") or (a," isn't in my_tuple"))
```

Comment:
even though it works this is not very pythonic. the goal of python is to find the simplest form of expressing the same thing. if it's boring you're on the right place.